create-project.js

```diff
@@ -1,12 +1,15 @@
 import { randomUUID } from 'crypto';
 import { createWriteStream, existsSync, mkdirSync, readFileSync, renameSync, unlinkSync, writeFileSync } from 'fs'
+import fsExtra from 'fs-extra'
```

Comment:
Should be able to add `copyFileSync` to the `fs` import and delete line 3 where you add the `fs-extra` import.

Reply:
There is a `cpSync` that does handles copying directories so have used that and removed the fs-extra dependency.